node.go

```diff
@@ -61,7 +61,7 @@ func (node *NodeBase) error(err error) {
 }
 
 func (node *NodeBase) errorf(format string, v ...interface{}) {
-	panic(fmt.Errorf("Jet Runtime Error(%q:%d): %s", node.TemplateName, node.Line, fmt.Sprintf(format, v...)))
+	panic(fmt.Errorf("jet runtime error @ %q:%d: %s", node.TemplateName, node.Line, fmt.Sprintf(format, v...)))
```

Comment:
I don't think there's a reason to change this? The previous version was fine. :)

Reply:
True, I changed it to lowercase to be similar to usual Go error messages, but it's not needed. https://github.com/CloudyKit/jet/pull/126/commits/a04ce10e0ad8891da46a2d625248df24e58133c4